src/99-appendices/11-qmri.md

```diff
@@ -0,0 +1,558 @@
+|----------------------------|-----------------------------------|--------------------------------------------|---------------------------------------------|
+| VFA                        | `PulseSequenceType` == `SPGR`     |                                            | DESPOT1                                     |
+| VFA                        | `PulseSequenceType` == `SSFP`     | `SpoilingRFPhaseIncrement` (`fixed`)       | DESPOT2                                     |
+| MP2RAGE                    |                                   | `EchoTime` (`echo`)                        | MP2RAGE-ME                                  |
+| MPM                        |                                   | `EchoTime` (`echo`)                        | MPM-ME                                      |
+
+<sup>*</sup> (`entity`/`fixed`) denotes whether the OPTIONAL metadata that forms a new
+flavor of qMRI applicaiton for the respective suffix varies across files of a file colleciton
+(which calls for using a linking entity) or fixed. If former is the case, the entity is to be
+added to the files in that file collection. Note that this addition MUST be allowed by the
+priority levels given for that suffix in the [`entity table`](./04-entity-table.md). If latter (`fixed`) is the case,
+filenames will remain the same; however, the optional metadata (third column) may
+define the flavor of the application (fourth column) along with the conditional value of a
+required metadata field (second column).
+
+A derived qMRI application becomes avaiable if all the optional metadata fields
+listed for the respective file collection suffix are provided for the data. In addition,
+conditional rules based on the value of a given required metada field can be set
+for the description of a derived qMRI application. Note that the value of this
+required metadata is fixed across constituent images of a file collection and defined
+in [this section](#Method-specific-priority-levels-for-qMRI-file-collections).
+
+For example, if the optional metadata field of `PulseSequenceType` is SPGR
+for a collection of anatomical images listed by the `VFA` suffix, the data
+qualifies for `DESPOT1` T1 fitting. For the same suffix, if the `PulseSequenceType`
+metadata field has the value of `SSFP`, and the `SpoilingRFPhaseIncrement` is
+provided as a metadata field, then the dataset becomes eligible for `DESPOT2`
+T2 fitting application.
+
+Please note that optional metadata fields listed in the [deriving the intended qMRI
+application from an ambiguous file collection table](#Deriving-the-intended-qMRI-application-from-an-ambiguous-file-collection) are included in the optional (third)
+column of [the priority levels table](#Method-specific-priority-levels-for-qMRI-file-collections) for the consistency of this appendix.
+
+## Introducing a new qMRI file collection
+
+If a qMRI application can not be interpreted as subset of an already existing suffix
```

Comment:
can not --> cannot

Reply:
"as subset" --> "as a subset" (although I think "subtype" might be better, WDYT?)

Reply:
35f6375 a683f40